kubernetes/README.md

```diff
@@ -0,0 +1,78 @@
+## 1. install flex volume driver on every linux agent node
+```
+sudo apt install jq -y
+
+sudo mkdir -p /etc/kubernetes/volumeplugins/azure~blobfuse/bin
+cd /etc/kubernetes/volumeplugins/azure~blobfuse/bin
+sudo wget -O blobfuse https://raw.githubusercontent.com/andyzhangx/Demo/master/linux/flexvolume/blobfuse/binary/ubuntu1604-4.4.0-104-generic/blobfuse
+sudo chmod a+x blobfuse
+
+cd /etc/kubernetes/volumeplugins/azure~blobfuse
+sudo wget -O blobfuse https://raw.githubusercontent.com/andyzhangx/Demo/master/linux/flexvolume/blobfuse/blobfuse
+sudo chmod a+x blobfuse
+```
+#### Note:
+Make sure `jq` package is installed on every node.
```

Comment:
nit: we already install jq as per step 1
nit 2: We need a note to tell the user that this is only for debian distros